ui-tests/src/test/resources/features/soak-deployments/maxIntegrations.feature

```diff
@@ -0,0 +1,629 @@
+# @sustainer: mkralik@redhat.com
+
+@ui
+@oauth
+@long-running
+@max-integrations
+Feature: Test maximum integration
```

Comment:
will this scenario be executed every time the test suite is executed? Is this not planned to be more a soak-test - executed only on demand?

Reply:
Yes. Yes, we can @disabled it and run it only on demand. Or only with the prod builds.

Reply:
@tplevko In order to don't miss any issues, I left `basic` scenario running during the nightly run and for `complex` scenario, I have used @prod annotation so that scenario is running only during the prod build testing. WDYT?

Reply:
@avano WDYT about usage @prod annotation in this case ?

Reply:
if you plan to run it `always` with prod build, then it makes sense

Reply:
@avano I plan to run it during the full testing cycle which happens only with prod build. Since this test is not marked with @smoke , it shouldn't run with the prod smoke and the other runs, should it?

Reply:
no :)